manifests/config.pp

```diff
@@ -30,4 +30,6 @@
     mode   => '0755',
   }
 
+  $services_real = prefix($katello::services_list, $katello::services_directory)
+  ensure_resource('file', $services_real, {'ensure' => 'file', 'mode' => '0644'})
```

Comment:
They're both `ensure_resource`.

 `katello-service` tool will look services.d and restart all the services by the filename.  It's simple, but your comment makes me realize this approach doesn't work.

If you create say, `/etc/katello/services.d/pulp` on a capsule and later remove the service, the directory's not going to get cleared (and can't, it's managed by two modules).

I'll find a different approach